src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -958,6 +1126,23 @@ type TableContext =
 /// </summary>
 module Scripting =
 
+    /// Factory method that allows one to include auto-initialization easily for scripting scenarios
+    type TableContext internal () =
+
+        /// <summary>Creates a DynamoDB client instance for the specified F# record type, client and table name.</summary>
+        /// <param name="client">DynamoDB client instance.</param>
+        /// <param name="tableName">Table name to target.</param>
+        /// <param name="throughput">Optional throughput to configure if the Table does not yet exist.</param>
+        /// <param name="metricsCollector">Function to receive request metrics.</param>
+        static member Initialize<'TRecord>
+            (   client : IAmazonDynamoDB, tableName : string, ?throughput,
+                ?metricsCollector : RequestMetrics -> unit) : TableContext<'TRecord> =
```

Comment:
Yes, I probably like the idea of `Create` and `Verify` better than this - it’s not particularly obvious here that you trigger 'create' behaviour by supplying throughput. I would hope the fact it’s taking an `IAmazonDynamoDB` client is enough of a hint that it’s not an in-memory table, if you really can’t be bothered reading the tooltip. Happy to drop metrics off the convenience constructors.

Reply:
Accidentally using Create instead of Verify in a script is not likely to be a big deal in practice - I guess only if you have a typo in the table name?

Reply:
>I would hope the fact it’s taking an IAmazonDynamoDB client is enough of a hint that it’s not an in-memory table, if you really can’t be bothered reading the tooltip. Happy to drop metrics off the convenience constructors.

My point was that the well known name `Create` in .NET does not imply `TableIfNotExists` ordinarily, even for a DB client - all sorts of things from Tuple.Create through all sorts of contexts for ORMs etc.

I agree the supplying of `throughput` triggering the behavior is cryptic though.

If we're not removing the TableAsync from the other two, maybe this can simply split into `Verify` and `VerifyOrCreate` ?

Removing metrics

Reply:
Hmm, maybe 2 overloads of `Initialize` is the least bad option. I didn’t mind `Create` implying static factory because that’s ultimately what it is (and that’s what it does now) but the naming collision will be painful & I feel it’s probably less confusing to keep this different to the non-scripting instance initialisation methods.

Reply:
Done.